conjure-java-verifier/build.gradle

```diff
@@ -0,0 +1,131 @@
+
+    generator 'com.palantir.conjure.java:conjure-java'
+
+    generatedObjectsCompile 'com.palantir.conjure.java:conjure-lib'
+    generatedJerseyCompile 'com.palantir.conjure.java:conjure-lib'
+    generatedJerseyCompile sourceSets.generatedObjects.output
+    generatedRetrofitCompile 'com.squareup.retrofit2:retrofit'
+    generatedRetrofitCompile 'com.palantir.conjure.java:conjure-lib'
+    generatedRetrofitCompile sourceSets.generatedObjects.output
+
+    testCompile 'junit:junit'
+    testCompile 'org.assertj:assertj-core'
+    // testCompile 'org.mockito:mockito-core'
+    testCompile project(':conjure-java-jaxrs-client')
+    testCompile project(':conjure-java-retrofit2-client')
+    testCompile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+    testCompile 'com.palantir.conjure.java:conjure-lib'
+    testCompile 'org.apache.commons:commons-lang3'
+    testCompile sourceSets.generatedObjects.output
+    testCompile sourceSets.generatedJersey.output
+    testCompile sourceSets.generatedRetrofit.output
+
+    // processor 'org.immutables:value'
+}
+
+tasks.withType(JavaCompile).configureEach {
+    options.errorprone {
+        check('Slf4jLogsafeArgs', net.ltgt.gradle.errorprone.CheckSeverity.OFF)
+    }
+}
+
+task unpackVerificationServer(type: Sync) {
+    from { tarTree(configurations.verificationServer.singleFile) }
+    into "${buildDir}/verification-server"
+    rename { "server" }
+}
+
+task unpackGenerator(type: Sync) {
+    from { tarTree(configurations.generator.singleFile) }
+    into "${buildDir}/generator"
+    ext.getExecutable = {
+        fileTree("${buildDir}/generator/${configurations.generator.singleFile.name.replace('.tgz', '')}/bin")
+            .filter({ !it.name.contains('.bat') })
+            .singleFile
+    }
+}
+
+task copyTestCases(type: Sync) {
+    from configurations.testCases, {
+        rename { "test-cases.json" }
+    }
+    from configurations.verificationApi, {
+        rename { "verification-api.json" }
+    }
+    into "$buildDir/test-cases"
+}
+
+task conjureJavaObjects(type: Exec, dependsOn: unpackGenerator) {
+    executable "${-> unpackGenerator.getExecutable()}"
+    args 'generate',  "${-> configurations.verificationApi.singleFile}", 'src/generatedObjects/java', '--objects'
+
+    inputs.file "${-> configurations.verificationApi.singleFile}"
+    outputs.dir "src/generatedObjects/java"
+    doFirst { delete "src/generatedObjects/java/*" }
+}
+
+task conjureJavaJersey(type: Exec, dependsOn: unpackGenerator) {
+    executable "${-> unpackGenerator.getExecutable()}"
+    args 'generate',  "${-> configurations.verificationApi.singleFile}", 'src/generatedJersey/java', '--jersey'
+
+    inputs.file "${-> configurations.verificationApi.singleFile}"
+    outputs.dir "src/generatedJersey/java"
+    doFirst { delete "src/generatedJersey/java/*" }
+}
+
+task conjureJavaRetrofit(type: Exec, dependsOn: unpackGenerator) {
```

Comment:
why do we generate retrofit?

Reply:
we want to test it!

Reply:
actually we should test it in conjure-java as well